modules/openapi-generator/src/main/resources/python-nextgen/api_client.mustache

```diff
@@ -563,7 +563,7 @@ class ApiClient(object):
             else:
                 new_params.append((k, v))
 
-        return "&".join(["=".join(item) for item in new_params])
+        return "&".join(["=".join(str(item)) for item in new_params])
```

Comment:
can you please elaborate on the fix for this line? can you share a spec to more easily reproduce the issue fixed by this change?